comp/prizes/acquire-prizes-trophies.yaml

```diff
@@ -0,0 +1,39 @@
+summary: Organise trophies for the competition
+
+priority: should
+
+component: Competition
+
+milestone: $SRYYYY Competition
+
+area-owner: production
+
+description: >-
+  The prizes for the competition are a combination of trophies and edible prizes.
+  This task only considers the trophies; there is a separate task for edible prizes.
```

Comment:
Not sure if this is easy to do, but if possible, it would be good to link to the edible prize task here.

Reply:
Unfortunately that's not possible right now. I have wanted this in the past though, so adding it would be a useful feature. One approach I've considered is to have links to "related" tasks.

A workaround for now might be to link to the template task (i.e: in the recurring tasks repo) instead.